### PR TITLE
code refactoring proposal

### PR DESCRIPTION
**Pull request by olekstomek**
Code refactoring proposal

### CODE REVIEW

File: src/components/Card.js [hunk 1]
```diff
@@ -234,3 +216,10 @@ export const Card = ({ direction = 'ltr' }) => {
     </div>
   );
 };
+
+function setEncryptionMode(network, e) {
```
Comment (PushpenderSaini0) [inline src/components/Card.js:220]: Convert to an ES6 function
and no need to pass network as an input parameter , as its already global and we will need it always
```JS
const setEncryptionMode = e => {
    ...network,
    encryptionMode: e.target.value,
  };
```

Or you could call setNetwork here
```JS
const setEncryptionMode = e => setNetwork({
    ...network,
    encryptionMode: e.target.value,
  });
```
Then you can change this

```diff
- onChange={(e) => setNetwork(setEncryptionMode(network, e))}
+ onChange={setEncryption}
```

Comment (olekstomek) [reply src/components/Card.js]: Thank you! I improved the code and now it is ES6.

**Review by PushpenderSaini0 — APPROVED** [review]

File: src/components/Card.js [hunk 2]
```diff
@@ -86,13 +95,7 @@ export const Card = ({ direction = 'ltr' }) => {
...
           <QRCode
             className="qrcode"
-            style={
-              !portrait
-                ? direction === 'ltr'
-                  ? { paddingRight: '1em' }
-                  : { paddingLeft: '1em' }
-                : {}
-            }
+            style={!portrait ? checkDirectionAndSetPadding : {}}
```
Comment (bndw) [inline src/components/Card.js:98]: Since you don't need the second case:
```suggestion
            style={!portrait && checkDirectionAndSetPadding}
```

**Review by bndw — APPROVED** [review]

File: src/components/Card.js (hunk 2 above)
Comment (olekstomek) [reply src/components/Card.js]: You're right. Commit

**Pull request merged**